### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by JTtNinjaCode** [JTtNinjaCode/Github-Feature-Test#1]
### Version/Branch of Dear ImGui:

Version 1.XX, Branch: XXX (master/docking/etc.)

### Back-ends:

imgui_impl_XXX.cpp + imgui_impl_XXX.cpp

### Compiler, OS:

Linux

### Full config/build information:

Test

### Details:

**My Issue/Question:**

XXX _(please provide as much context as possible)_

### Minimal, Complete and Verifiable Example code:

```cpp
// Here's some code anyone can copy and paste to reproduce your issue
ImGui::Begin("Example Bug");
MoreCodeToExplainMyIssue();
ImGui::End();
```

### your favoriate programming language

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by JTtNinjaCode** [comment]
Hi, this is a saved replies test

**Comment by JTtNinjaCode** [comment]
TL;DR
<details><summary>Details</summary>
<p>

Hidden Text

</p>
</details>

**Comment by JTtNinjaCode** [comment]
[test.txt](https://github.com/user-attachments/files/17528005/test.txt)

**Comment by JTtNinjaCode** [comment]
link to this issue: #1
